sql/moz-fx-data-shared-prod/fivetran_costs_derived/monthly_connector_costs_v1/query.sql

```diff
@@ -25,44 +25,66 @@ total_mar AS (
     destination_id,
     measured_month
 ),
-monthly_connector_costs AS (
+total_mar AS (
   SELECT
-    destinations.destination_name,
+    measured_month,
+    SUM(paid_active_rows) AS paid_active_rows,
+  FROM
+    destinations_total_mar
+  GROUP BY
+    measured_month
+),
+monthly_connector_ratios AS (
+  SELECT
+    destination_id,
     measured_month,
     connector_mar.connector_name,
     connector_mar.paid_active_rows,
     connector_mar.free_active_rows,
     connector_mar.total_active_rows,
+    ROUND(
+      connector_mar.paid_active_rows / NULLIF(destinations_total_mar.paid_active_rows, 0) * 100,
+      2
+    ) AS percentage_of_destination_paid_active_rows,
     ROUND(
       connector_mar.paid_active_rows / NULLIF(total_mar.paid_active_rows, 0) * 100,
       2
-    ) AS percentage_of_total_paid_mar,
+    ) AS percentage_of_total_active_rows
```

Comment:
The `paid` part of the name got removed, but it'd probably be good to keep that in for clarity.